### ISSUE TITLE
Regression: codeFrame formatter isn't allowed now as a linter format

### ISSUE THREAD

**Issue by MikaAK** [angular/angular-cli#11046]
### Versions
```
Angular CLI: 6.0.5
Node: 9.3.0
OS: darwin x64
Angular: 6.0.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.5
@angular-devkit/build-angular     0.6.5
@angular-devkit/build-optimizer   0.6.5
@angular-devkit/core              0.0.29
@angular-devkit/schematics        0.0.52
@angular/cdk                      6.1.0
@angular/cli                      6.0.5
@angular/flex-layout              2.0.0-beta.12
@angular/material                 6.1.0
@ngtools/json-schema              1.1.0
@ngtools/webpack                  6.0.5
@schematics/angular               0.6.5
@schematics/update                0.6.5
rxjs                              6.2.0
typescript                        2.8.3
webpack                           4.8.3
```

### Repro steps

* Step 1 Update from old version of cli (was on 1.6.1)
* Step 2 Try and run ng lint with `ng lint client --format=codeFrame

### Observed behavior
```
Schema validation failed with the following errors:
  Data path ".format" should be equal to one of the allowed values.
```
### Desired behavior
Should use tslint `codeFrame` formatter

### Mention any other details that might be useful (optional)
This didn't used to error on prior versions

**Comment by clydin** [comment]
Resolved via #10904 and will be in the next patch release.

**Comment by zakhenry** [comment]
@clydin is this actually fixed? I still get the following error when trying to use codeFrame
```
Schema validation failed with the following errors:
  Data path ".format" should be equal to one of the allowed values.
```
